### PR TITLE
DOC: improved the docstring of pandas.Index.min()

### PR DESCRIPTION
**Pull request by juanhuguetgarcia**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [X] PR title is "DOC: update the <your-function-or-method> docstring"
- [X] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [X] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [X] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [X] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
Docstring for "pandas.Index.min" correct. :)
```

If the validation script still gives errors, but you think there is a good reason
to deviate in this case (and there are certainly such cases), please state this
explicitly.

### CODE REVIEW

**Comment by caperutxa** [comment]
Change it

File: pandas/core/base.py [hunk 1]
```diff
@@ -801,7 +801,31 @@ def argmax(self, axis=None):
         return nanops.nanargmax(self.values)
 
     def min(self):
-        """ The minimum value of the object """
+        """
+        Return the minimum value of the Index object.
+
+        Return the maximum value of the object within the same type.
```
Comment (arnau126) [inline pandas/core/base.py]: minimum, not maximum

Comment (juanhuguetgarcia) [reply pandas/core/base.py]: thanks

**Comment by caperutxa** [comment]
Looks fine, as index.max ;)

File: pandas/core/base.py [hunk 2]
```diff
@@ -801,7 +801,31 @@ def argmax(self, axis=None):
...
+        """
+        Return the minimum value of the Index object.
+
+        Return the minimum value of the object within the same type.
+        The Index of a DataFrame can be accessed as pandas.DataFrame.index.
+
+        Returns
+        -------
+        scalar or object
+            minimum value
```
Comment (villasv) [inline pandas/core/base.py]: The description must start with a capital letter, and finish with a dot.

File: pandas/core/base.py [hunk 3]
```diff
@@ -801,7 +801,31 @@ def argmax(self, axis=None):
...
+        """
+        Return the minimum value of the Index object.
+
+        Return the minimum value of the object within the same type.
+        The Index of a DataFrame can be accessed as pandas.DataFrame.index.
+
+        Returns
+        -------
+        Scalar or object.
+            minimum value
```
Comment (villasv) [inline pandas/core/base.py]: Sorry, perhaps I wasn't clear before.
```
scalar or object
            Minimum value.
```

I'll quote the docs:

> The documentation of the return is also similar to the parameters. But in this case, no name will be provided, unless the method returns or yields more than one value (a tuple of values).

> The parameters are defined by their name, followed by a space, a colon, another space, and the type (or types). Note that the space between the name and the colon is important. Types are not defined for *args and **kwargs, but must be defined for all other parameters. After the parameter definition, it is required to have a line with the parameter description, which is indented, and can have multiple lines. The description must start with a capital letter, and finish with a dot.

**Review by villasv — CHANGES_REQUESTED** [review]

File: pandas/core/base.py (hunk 3 above)
Comment (juanhuguetgarcia) [reply pandas/core/base.py]: I think now should be fine, thanks!

**Comment by TomAugspurger** [comment]
Thanks everyone!

**Pull request merged**